deepmd/pd/model/network/init.py

```
@@ -0,0 +1,458 @@
+            recommended to use only with ``'relu'`` or ``'leaky_relu'`` (default).
+        generator: the paddle Generator to sample from (default: None)
+        reverse (bool, optional): Tensor data format order, False by default as
+            [fout, fin, ...].. Defaults to False.
+
+    Examples
+    --------
+        >>> w = paddle.empty(3, 5)
+        >>> nn.init.kaiming_uniform_(w, mode="fan_in", nonlinearity="relu")
+    """
+    if 0 in tensor.shape:
+        warnings.warn("Initializing zero-element tensors is a no-op")
```

Ruff

No explicit `stacklevel` keyword argument found

(B028)